README.md

```diff
@@ -34,6 +34,7 @@
 ```javascript
 // Imports the Google Cloud client library
 const sc = require('@google-cloud/securitycenter');
+console.log(sc);
```

Comment:
don't think we want this :)

Reply:
Added some quickstart: https://github.com/googleapis/nodejs-security-center/pull/33